website/pages/docs/builders/proxmox.mdx

```diff
@@ -213,6 +213,29 @@ builder.
 - `cloud_init_storage_pool` - (string) - Name of the Proxmox storage pool
   to store the Cloud-Init CDROM on. If not given, the storage pool of the boot device will be used.
 
+- `additional_iso_files` (array of objects) - Additional ISO filess attached to the virtual machine.
+  Example:
+
+  ```json
+  [
+    {
+      "device": "scsi",
+      "bus_number": "3",
+      "filemane": "local:iso/virtio-win-0.1.185.iso",
+      "unmount": true
+    }
+  ]
+  ```
+  - `device` (string) - Bus type that the ISO will be mountet on. Can be `ide`,
```

Comment:
Typo
```suggestion
  - `device` (string) - Bus type that the ISO will be mounted on. Can be `ide`,
```

Reply:
Also, wouldn't it semantically be a `device_type` or similar, since a `device` would include the number as well?